teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -90,49 +90,110 @@ With regards to this checklist, reviewers should do the following:
+
+![Micro &#x2014;&amp;gt; Macro!](../../../.gitbook/assets/smalltobig.jpeg)
+
+**Images**! Make liberal use of images \(like the one above\) in your cards. It's much easier to visualize a concept than to read three paragraphs of text. Obviously, text is necessary, but pairing text with visuals is immensely helpful. Also, images help break up blocks of text, making it easier on the eyes. Please use copyright-free images, Pexels is a great source. Make sure images are embedded using &lt;img&gt; tags and not URLs.
+
+Activities and Labs will feature coding tasks for students to complete. For these, developers should implement what we call **checkpoints**—Python console output that serves as a frame of reference for students to make sure their code is doing what it's supposed to be doing. 
+
+**Titles**. These should be included in HTML comments at the top of each card, not in the file name itself \(remember, cards are named numerically\). Along with titles, **concepts** covered in the card should also be contained in HTML comments. Here's an example:
+
+![Titles an concepts love swimming in HTML comments.](../../../.gitbook/assets/screen-shot-2020-03-30-at-9.48.40-am.png)
+
+**Card length** should be reasonable. Ideally, keep the entire card viewable with at most one scroll.  
+Students will have a hard time staying focused if a card is too long, and shorter cards are easier to digest. Try to split up topics as much as possible. 
+
+This one is pretty self-explanatory, but make sure your cards progress in a way that makes sense. 
 
 #### Stage 1 - Lab Addendum
 
-* [ ] Hard cards provide enough guidance for the student to finish task
-* [ ] Mediums bridge Hards and Easys
-* [ ] Easy cards provide solution, are line-separated
-* [ ] Solution and starter code in Airtable and GitHub
-  * [ ] Code is testable, not too general
+![Not a paid endorsement, this picture just fits perfectly.](../../../.gitbook/assets/inthelab.jpg)
+
+When creating a Lab, you are essentially giving the student a problem to solve, breaking it down into steps, with each step subdivided into cards of hard, medium, and easy difficulty. An example might be to code a Python program that can solve Sudoku. 
+
+When it comes to Labs, there are a number of things to keep in mind.
+
+**Hard cards** are, by definition, _hard._ A hard card for the Sudoku Solver Lab might explain the game of Sudoku and have a teeny-tiny bit of **starter code** at the end. Keep in mind that, although this card should stretch the student's brain, it should also provide just enough guidance for the student to finish the task. The main thing to keep in mind is you _should NOT_ leak any hints_._ Example [here](https://github.com/bitprj/curriculum/blob/master/Data-Structures-and-Algos-Topic/labs/Lab6_Sudoku_Solver/Cards/1.md). 
```

Comment:
Hard cards should give all the information they need to complete the problem.'

Include the fact that each hard card will make reference to a function header or other code section given in the starter code, providing guidance on what functionalities should be implemented at each hard card.